packages/myst-cli/src/transforms/embed.ts

```diff
@@ -16,20 +17,31 @@ export function embedDirective(
   dependencies: Dependency[],
   state: IReferenceState,
 ) {
-  const embedNodes = selectAll('embed', mdast) as GenericNode[];
+  const embedNodes = selectAll('embed', mdast) as Embed[];
   embedNodes.forEach((node) => {
     const normalized = normalizeLabel(node.label);
```

Comment:
Put label in embed `source`, confusing that label/identifier are references not targets in this case.